inc/search.class.php

```diff
@@ -5616,15 +5605,27 @@ static function giveItem($itemtype, $ID, array $data, $meta = 0,
 
                         if ($itemtype == 'Ticket') {
                            if (isset($data[$ID][$k]['name'])
-                                 && $data[$ID][$k]['name'] > 0) {
-                              $userdata = getUserName($data[$ID][$k]['name'], 2);
-                              $tooltip  = "";
-                              if (Session::haveRight('user', READ)) {
-                                 $tooltip = Html::showToolTip($userdata["comment"],
-                                                              ['link'    => $userdata["link"],
-                                                                    'display' => false]);
+                              && $data[$ID][$k]['name'] > 0
+                           ) {
+                              if (Session::getCurrentInterface() == 'helpdesk'
+                                 && $orig_id == 5 // -> Assigned user
+                                 && !empty($anon_name = User::getAnonymizedName(
+                                    2,
```

Comment:
```suggestion
                                    $data[$ID][$k]['name'],
```

Reply:
Good catch, I indeed forgot to put the correct var.
-> Done in 7698dc4.